services/spar/test-integration/Util/SCIM.hs

```diff
@@ -184,6 +211,11 @@ createUser_
 createUser_ auth user spar_ = do
     -- NB: we don't use 'mkEmailRandomLocalSuffix' here, because emails
     -- shouldn't be submitted via SCIM anyway.
+    -- TODO: what's the consequence of this?  why not update emails via
+    -- SCIM?  how else should they be submitted?  i think this there is
```

Comment:
Allowing SCIM to write any kind of unvalidated email addresses to our database seems dangerous to me, regardless of whether they go in Brig or in Spar. (I don't think I'm confusing validated and SCIM-provided emails, although of course I can't be fully sure about that.)

Even though the current code doesn't do anything bad with emails, I'm still worried because we don't document our design decisions and – if we won't be around in the future for some reason or other – I think it's not unlikely that the next person who has to touch Spar _will_ commit some major blunders.

I have spent the last ten minutes going around the room and thinking about a way to perform hostile account takeover if somebody e.g. implements a Spar-to-Brig sync (or a DB migration script) and forgets about email validation. I admit that I haven't found any, but I also haven't managed to even remotely convince myself that it's impossible.

Unless there's a good reason not to forbid incoming emails (IIRC Okta can be configured not to send them), I would strongly prefer that we don't allow them in user creation requests. Unless I'm mistaken, it doesn't require giving up on the whole "store users in the database verbatim" thing – we don't have to wipe emails, we just have to disallow them.